Sources/Pageboy/Components/PageboyAutoScroller.swift

```diff
@@ -45,6 +45,7 @@ public class PageboyAutoScroller: Any {
     public enum IntermissionDuration {
         case short
         case long
+        // swiftlint:disable identifier_name
```

Comment:
I think this error can be fixed by upgrading SwiftLint? I definitely remember seeing this under Xcode 10 / Swift 4.2 and don't anymore on `0.27.0`.

Reply:
Yeah that fixed it, thanks